app/views/wiki/_header.html.erb

```diff
@@ -31,7 +31,13 @@
 
     <div style="padding-top:8px;text-align:center">
       <span class="d-none d-lg-inline">
-        <span rel="tooltip" title="The number of authors for this page." data-placement="top"><i style="color:#888;" class="fa fa-user"></i> <%= number_with_delimiter(@node.authors.size) %></span>  |
+        <span rel="popover" style='cursor: pointer;' title="Our Authors" data-html="true" data-placement="top" data-content= "
```

Comment:
Hi @Priyaraj17, sorry I didn't see the issue link on the bottom before...maybe we could edit the title to just `Authors` what do you think? Thanks

Reply:
Yeah sure.

Reply:
I have made the changes as requested.